services/minespace-web/src/components/pages/Incidents/IncidentSuccessPage.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+import { Row, Col, Typography, Button, Divider } from "antd";
+import { ArrowLeftOutlined, CheckCircleOutlined } from "@ant-design/icons";
+import PropTypes from "prop-types";
+import * as routes from "@/constants/routes";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  location: PropTypes.shape({
+    state: PropTypes.shape({
+      incident: CustomPropTypes.incident,
+    }),
+  }).isRequired,
+};
+
+export const IncidentSuccessPage = (props) => {
+  const renderContent = () => {
+    const incident = props.location.state?.incident || {};
+    const { mine_incident_guid, mine_guid, mine_determination_type_code } = incident;
+    return (
+      <div style={{ textAlign: "center" }}>
+        <>
+          <Row>
+            <Col span={24}>
+              <CheckCircleOutlined
+                style={{
```

Comment:
I notice IRT and Major Mines Application success pages also use this same style. Does it make sense to create a class for it at this point? Not sure if we are going to use have other features using this pattern moving forward, but there may be a chance of it. Dealer's choice.